src/pudl/extract/eia860.py

```diff
@@ -65,7 +65,7 @@ def get_dtypes(page, **partition):
     "raw_boiler_info_eia860",
     "raw_boiler_mercury_eia860",
     "raw_boiler_nox_eia860",
-    "raw_boiler_pm_eia860",
+    "raw_boiler_particulate_eia860",
```

Comment:
It seems like neither `raw_boiler_particulate` nor `raw_boiler_pm` are being used anywhere, but we're putting `raw_boiler_particulate` into the outputs of `eia_transform` and not putting `raw_boiler_pm` in - so I swapped these out.

In general, what's the best way to see if something has downstream deps? Look in dagit?

Reply:
Not sure how to answer your second question, but in terms of the first yes that is correct, these tables aren't getting used anywhere (they will eventually get used in a transform that has yet to be written).

Reply:
I think dagit is the best method for viewing dependencies. I'll check if there is a way to check programmatically.